packages/core/src/lib/toast/toast-container.svelte

```diff
@@ -11,11 +11,11 @@ It needs access to context from `provideToast`, which should also
 be added to the root layout component.
 -->
 <script lang="ts">
-import { fade, fly } from 'svelte/transition';
-import { flip } from 'svelte/animate';
+import { fly, fade } from 'svelte/transition';
 
 import { useToastState } from './context';
 import ToastBanner from './toast-banner.svelte';
+import { flip } from 'svelte/animate';
```

Comment:
[Nit] Why was this import statement moved?

Reply:
That was unintentional